bigtable/beam/helloworld/src/test/java/HelloWorldTest.java

```diff
@@ -98,13 +100,21 @@ public static void afterClass() {
     }
   }
 
+  /**
+   * Test the writing data example as well as runtime dependency building. This must be run via
```

Comment:
I would reword slightly:

"This is tested via dataflow to ensure there are no dependency conflicts which can cause runtime errors."